version/version.go

```diff
@@ -0,0 +1,14 @@
+package version
+
+import "runtime/debug"
+
+var Commit = func() string {
+	if info, ok := debug.ReadBuildInfo(); ok {
+		for _, setting := range info.Settings {
+			if setting.Key == "vcs.revision" {
+				return setting.Value
+			}
+		}
+	}
+	return ""
```

Comment:
nit: even if it duplicates the `return ""` line, a guard error clause makes the code more readable by indenting the non-happy paths.

```suggestion
	info, ok := debug.ReadBuildInfo()
	if !ok {
		return ""
	}

	for _, setting := range info.Settings {
		if setting.Key == "vcs.revision" {
			return setting.Value
		}
	}

	return ""
```